core/ui.py

```diff
@@ -304,9 +304,10 @@ def CurrentLocation(self):
     else:
       return runtime.NO_SPID
 
-  def PrefixPrint(self, msg, prefix, span_id=runtime.NO_SPID):
-    # type: (str, str, int) -> None
+  def PrefixPrint(self, msg, prefix, blame_loc):
+    # type: (str, str, loc_t) -> None
     """Print a hard-coded message with a prefix, and quote code."""
+    span_id = location.GetSpanId(blame_loc)
```

Comment:
we can use our function to convert a loc_t to a span_id

Later will remove it from this internal function, but for now it lets us make a small diff